### DISCUSSION TITLE
Updating lensfun On Linux Mint

### DISCUSSION THREAD

**Discussion by MitchWoodin** [lensfun/lensfun#2103] [Q&A]
Hey,

I'm trying to update lensfun for darktable on Linux Mint 21.
When I try and run lensfun-update-data I get this 

![image](https://github.com/lensfun/lensfun/assets/42667068/6777ba5d-0132-4ccc-a323-cdb263fffe27)

I'm not entirely sure where to go from here

Thanks,
Mitchell

**Comment by sarunasb** [comment]
Python module `lensfun` is provided by `python3-lensfun` package. `liblensfun-bin`, which provides `lensfun-update-data`, should have pulled it as a dependency...

**Comment by MitchWoodin** [comment]
Hey, 
This is what I get when I try and install those packages:
![image](https://github.com/lensfun/lensfun/assets/42667068/44f1dcc6-3189-4a53-b685-1d556fc3b06e)

Any ideas?

**Comment by sarunasb** [comment]
Something to do with your user's Python setup. 

It's OK to update Lensfun data system-wide: `sudo lensfun-update-data`.

P.S. If possible, please use copy and paste, not screen capture images — text in those will not be searchable and some of it is 
hardly readable.
